content/nginx-one/how-to/nginx-configs/view-edit-nginx-configurations.md

```diff
@@ -1,15 +1,30 @@
 ---
-docs: 
-doctypes:
-    - task
-tags:
-    - docs
+# We use sentence case and present imperative tone
 title: View and edit NGINX configurations
-toc: true
+# Weights are assigned in increments of 100: determines sorting order
 weight: 300
+# Creates a table of contents and sidebar, useful for large documents
+toc: true
+# Types have a 1:1 relationship with Hugo archetypes, so you shouldn't need to change this
+type: tutorial
+# Intended for internal catalogue and search, case sensitive:
+product: NGINX One
+# Intended for internal catalogue
+docs: "DOCS-000"
```

Comment:
Leaving this out completely serves the same purpose as adding DOCS-000, and is preferred.

Reply:
```suggestion

```